### PR TITLE
added many customizer settings for text, input, input focus, entry title...

### PR DESCRIPTION
**Pull request by petenelson**
..., etc, colors

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -78,8 +78,13 @@ function stock_setup() {
...
 	return array(
-		'primary'   => '#000000',
-		'secondary' => '#f5f5f5',
+		'primary'   			=> '#000000',
+		'secondary' 			=> '#f5f5f5',
+		'text' 					=> '#404040',
+		'input_background'		=> '#ffffff',
+		'input_text'			=> '#666666',
+		'input_focus'			=> '#222222',
+		'entry_title'			=> '#000000',
```
Comment (lukecarbis) [inline functions.php]: Please change these `tab` characters to spaces, aligned with the longest key.

See the _Exception_ listed here: https://make.wordpress.org/core/handbook/coding-standards/php/#indentation

File: functions.php [hunk 2]
```diff
@@ -117,9 +122,14 @@ function stock_scripts() {
...
+	$default_colors  			= stock_get_default_colors();
+	$primary_color   			= get_theme_mod( 'stock_primary_feature_color', isset( $default_colors['primary'] ) ? $default_colors['primary'] : null );
+	$secondary_color 			= get_theme_mod( 'stock_secondary_feature_color', isset( $default_colors['secondary'] ) ? $default_colors['secondary'] : null );
+	$text_color 	 			= get_theme_mod( 'stock_text_color', isset( $default_colors['text'] ) ? $default_colors['text'] : null );
+	$input_background_color		= get_theme_mod( 'stock_input_background_color', isset( $default_colors['input_background'] ) ? $default_colors['input_background'] : null );
+	$input_text_color			= get_theme_mod( 'stock_input_text_color', isset( $default_colors['input_text'] ) ? $default_colors['input_text'] : null );
+	$input_focus_color			= get_theme_mod( 'stock_input_focus_color', isset( $default_colors['input_focus'] ) ? $default_colors['input_focus'] : null );
+	$entry_title				= get_theme_mod( 'stock_entry_title_color', isset( $default_colors['entry_title'] ) ? $default_colors['entry_title'] : null );
```
Comment (lukecarbis) [inline functions.php]: See comment above.

File: functions.php [hunk 3]
```diff
@@ -137,7 +147,41 @@ function stock_scripts() {
 	}
 	h2.site-description {
 		color: {$secondary_color};
-	}";
+	}
+	body,
+	button {
+		color: {$text_color}
```
Comment (lukecarbis) [inline functions.php]: Another nitpick: please add a semicolon to the end of these properties (for consistency and future editability).

File: style.css [hunk 4]
```diff
@@ -4,7 +4,7 @@ Theme URI: https://github.com/lukecarbis/stock
 Author: Luke Carbis
 Author URI: http://carb.is
 Description: A simple theme, inspired by minimalism & precision.
-Version: 1.0.3
+Version: 1.1.0
```
Comment (lukecarbis) [inline style.css]: Nice. :smile_cat:

**Comment by petenelson** [comment]
Spacing and semicolons updated

File: js/customizer.js [hunk 5]
```diff
@@ -32,6 +32,46 @@
...
+		value.bind( function( to ) {
+			$( 'input, input[type=\"text\"], input[type=\"email\"],	input[type=\"url\"], input[type=\"password\"], input[type=\"search\"], select, textarea' ).css( {
+				'color': to,
+			} );
+		} );
+	} );
+	// Input focus text color.
+	wp.customize( 'stock_input_focus_color', function( value ) {
+		value.bind( function( to ) {
+			$( 'input:focus, input[type=\"text\"]:focus, input[type=\"email\"]:focus, input[type=\"url\"]:focus, input[type=\"password\"]:focus, input[type=\"search\"]:focus, select:focus, textarea:focus ' ).css( {
```
Comment (lukecarbis) [inline js/customizer.js]: This selector only selects the _currently focused_ elements. This means that this code won't actually work within the customiser.

We should do something like this instead:
```
wp.customize( 'stock_input_focus_color', function( value ) {
    var inputs = $( 'input, input[type=\"text\"], input[type=\"email\"], input[type=\"url\"], input[type=\"password\"], input[type=\"search\"], select, textarea' );
    value.bind( function( to ) {
        inputs.off( 'focus' ).on( 'focus', function() {
           $( this ).css( {
               'color': to,
           } );
        } );
       inputs.off( 'blur' ).on( 'blur', function() {
           $( this ).css( {
               'color': '',
           } );
        } );
    } );
} );

```

Comment (petenelson) [inline js/customizer.js]: Aha, that makes sense.  It wasn't working and couldn't figure out why.  I'll get this update tomorrow.

**Pull request merged**